content/en/integrations/guide/azure-architecture-and-configuration.md

```diff
@@ -0,0 +1,220 @@
+  - Ingest a range of both general and resource-specific metadata (including custom Azure tags), and apply it to the associated resource metrics in Datadog as tags
+  - Query Azure Metadata APIs and use the responses to [generate useful metrics in Datadog][2] for insights that Azure Monitor does not support
+
+The Azure APIs used and data collected are identical regardless of whether you use the standard or Azure Native version of the integration.
+
+{{% site-region region="us,us5,eu,gov,ap1" %}}
+
+### Standard Azure Integration Metric and Data Collection
+
+_Available in all Datadog Sites_
+
+Follow these steps to enable the standard Azure integration:
+
+  1. Create an app registration in your Active Directory and enter the credentials in the [Datadog Azure integration page][2].
+  2. Give the application read-access (`Monitoring Reader` role) to the subscriptions or management group you would like to monitor.
+
+The diagram below outlines the process and resulting architecture of the Azure Integration configuration described in the main documentation.
+
+{{< img src="integrations/guide/azure_architecture_and_configuration/app_registration_integration_setup.png" alt="Diagram of the App Registration integration setup" >}}
+
+Once this is completed, data collection begins automatically. The App Registration information entered in Datadog allows Datadog to [request a token from Azure Active Directory][1] (AD). Datadog uses this token as the authorization for API calls to various Azure APIs, to discover resources within the scope provided and collect data. This continuous process runs with two-minute intervals by default, and is used to discover and collect data from your Azure environment. This process is pictured below.
+
+{{< img src="integrations/guide/azure_architecture_and_configuration/app_registration_metric_collection.png" alt="Diagram of the App Registration integration setup" >}}
+
+[1]: https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/
+[2]: https://app.datadoghq.com/integrations/azure
+{{% /site-region %}}
+{{% site-region region="us3" %}}
+
+### Azure Native integration metric collection
+_Available only in the Datadog US3 site_
+
+Follow these steps to enable the Azure Native integration:
+
+1. Confirm that your Datadog organization is hosted on the US3 [Datadog site][1], or [create a trial Datadog account on the US3 site][5].
+2. Create a Datadog resource in Azure that links at least one subscription.
+3. Optionally, update the Datadog resource to include other subscriptions.
+
+The Datadog resource in Azure is a link between your Azure environment and your Datadog account. This link enables the same data collection as the standard Azure integration available for other Datadog sites, but with a different authentication mechanism. Its access is assigned using a **System Managed Identity** associated with the Datadog resource in Azure, rather than a user-created and configured **App Registration**.
```

Comment:
```suggestion
The Datadog resource in Azure links your Azure environment and your Datadog account. This link enables the same data collection as the standard Azure integration available for other Datadog sites, but with a different authentication mechanism. Its access is assigned using a **System Managed Identity** associated with the Datadog resource in Azure, rather than a user-created and configured **App Registration**.
```